Tools/scripts/install-prereqs-ubuntu.sh

```diff
@@ -20,12 +20,10 @@ version=$(lsb_release -r -s)
 yrelease=$(echo "$version" | cut -d. -f1)
 mrelease=$(echo "$version" | cut -d. -f2)
 
-if [ "$yrelease" -ge "$UBUNTU_YEAR" ]; then
```

Comment:
I guess this was my fault 😞: https://github.com/ArduPilot/ardupilot/pull/4979#discussion_r83275906

From what I remember though the idea was to install 3.0 starting with 15.10 and 2.8 on previous ones.
This change is bad as just checking the month is worthless. I think the correct fix is:

`if [[ ("$yrelease" -ge "$UBUNTU_YEAR" && "$mrelease" -ge "$UBUNTU_MONTH") || "$yrelease" -gt "$UBUNTU_YEAR" ]]; then`